src/VolleyManagement.Backend/VolleyManagement.Services/TeamService.cs

```diff
@@ -188,14 +189,29 @@ public Player GetTeamCaptain(Team team)
             return GetPlayerById(team.Captain.Id);
         }
 
+        /// <summary>
+        /// Change captain for existing team.
+        /// </summary>
+        /// <param name="team">team for editing.</param>
+        /// <param name="captainId">Player who should become captain.</param>
+        public void ChangeCaptain(TeamId team, PlayerId captainId)
+        {
+            _authService.CheckAccess(AuthOperations.Teams.Edit);
+            var teamToEdit = Get(team.Id);
```

Comment:
What if there is no instance with given Id?

Reply:
fixed